src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -2040,72 +1941,6 @@ public void T007_PackageProperties()
         }
 
         [Fact]
-        public void T006_CreateExternalRelationshipInvalidUri()
```

Comment:
Verified, it was the Uri throwing the error.

After a little more research, this test was maybe attempting to test for invalid use of an absolute Uri as an internal relationship.  I've changed it to reflect this possibility.